app/models/data_resources/tour.rb

```diff
@@ -5,46 +5,27 @@
 # the surrounding areas of the Smart Village
 #
 class Tour < Attraction
-  include FilterByRole
-
-  attr_accessor :force_create
   enum means_of_transportation: { bike: 0, canoe: 1, foot: 2 }
 
-  belongs_to :data_provider
-
   has_many :data_resource_categories, -> { where(data_resource_type: "Tour") }, foreign_key: :data_resource_id
   has_many :categories, through: :data_resource_categories
   has_many :geometry_tour_data, as: :geo_locateable, class_name: "GeoLocation", dependent: :destroy
+  has_many :tour_tour_stops, class_name: "TourTourStop", foreign_key: :tour_id, dependent: :destroy
```

Comment:
die Kreutzabelle müsste eigentlich tours_tour_stops heißen. Durch die Doppelung

```
has_and_belongs_to_many associative tables generally have the naming convention of [plural_first_alphabetical_model]_[plural_second_alphabetical_model] with any shared naming prepended
```

Finde ich durch die Doppelung mit Tour aber irgendwie auch nicht viel besser.
Was hältst du von `tour_stop_assignments`und demzufolge das Model auch TourStopAssignment

Reply:
I like the idea, done with bfe6031